packages/itwinui-css/backstop/tests/slider.html

```diff
@@ -61,674 +61,709 @@ <h1>Slider</h1>
+            style="inset-inline-start: 50%;"
+          ></div>
         </div>
-
       </div>
 
-      <br /><br />
-
-      <div class="iui-input-grid">
-        <div class="iui-input-label">Slider with ticks</div>
-
-        <div class="iui-slider-component-container iui-slider-horizontal">
-          <span class="iui-slider-min">0</span>
-          <div class="iui-slider-container iui-progress-slider">
-            <div class="iui-slider-rail"></div>
-            <div
-              class="iui-slider-track"
-              style="right: 50%;"
-            ></div>
-            <div
-              style="left: 50%;"
-              class="iui-slider-thumb"
-              role="slider"
-              tabindex="0"
-              aria-label="Percent Complete"
-              aria-valuemin="0"
-              aria-valuenow="50"
-              aria-valuemax="100"
-              id="test-slider-2"
-            ></div>
-            <output
-              class="iui-tooltip"
-              style="left: 50%;"
-            >
-              50
-            </output>
-            <div class="iui-slider-ticks">
-              <!-- You could generate the ticks based on your min, max & step values. -->
-              <span class="iui-slider-tick">0</span>
-              <span class="iui-slider-tick">10</span>
-              <span class="iui-slider-tick">20</span>
-              <span class="iui-slider-tick">30</span>
-              <span class="iui-slider-tick">40</span>
-              <span class="iui-slider-tick">50</span>
-              <span class="iui-slider-tick">60</span>
-              <span class="iui-slider-tick">70</span>
-              <span class="iui-slider-tick">80</span>
-              <span class="iui-slider-tick">90</span>
-              <span class="iui-slider-tick">100</span>
-            </div>
-          </div>
-          <span class="iui-slider-max">100</span>
+      <div
+        class="iui-slider-container"
+        data-iui-orientation="horizontal"
+      >
+        <div class="iui-slider iui-progress-slider">
+          <div
+            class="iui-slider-track"
+            style="inset-inline-end: 50%;"
+          ></div>
+          <div
+            class="iui-slider-thumb"
+            role="slider"
+            aria-label="Volume setting"
+            aria-orientation="horizontal"
+            aria-valuemin="0"
+            aria-valuenow="50"
+            aria-valuemax="100"
+            aria-disabled="true"
+            style="inset-inline-start: 50%;"
+          ></div>
         </div>
-
       </div>
 
-      <br /><br />
-
-      <div class="iui-input-grid">
-        <div
-          class="iui-input-label"
-          data-iui-disabled="true"
-        >Disabled slider</div>
-
-        <div class="iui-slider-component-container iui-slider-horizontal iui-disabled">
-          <span class="iui-slider-min">0</span>
-          <div class="iui-slider-container iui-progress-slider">
-            <div class="iui-slider-rail"></div>
```

Comment:
Jon and I spoke about this before, but just wanted to write a written comment too to confirm with other reviewers. Is it okay to change `iui-slider-rail` from an HTML element to a pseudo element? Because:

* In JS, the `.iui-slider::before` selector does not work since `::before` is not a real dom element.
* In React, I don't think we can add ref, props, etc. to the rail since this is not a real dom element.

Reply:
I don't think users would want or even should to customise rail + it can be customised by passing a class to wrapping element with ::before selector.

Reply:
The only rail customization I've seen so far is what's done by us in the color picker.  It can still be customized via `.myCustomClass::before` and I think that should be fine.